src/pages/index.js

```diff
@@ -1,140 +1,250 @@
+    })
+    .catch((err) => {
+      console.log(`Unable to process request, ${err}`);
+    })
+    .finally(() => {
+      profileEditModal.close();
+      profileEditModal.renderLoading("Save");
+    });
+}
+
+function handleAddCardSubmit(inputValues) {
+  addCardPopup.renderLoading("Loading...");
+  api
+    .createCard(inputValues)
+    .then((res) => {
+      renderCard(res);
+      addCardPopup.close();
+      addCardValidator.resetValidation();
+    })
+    .finally(() => {
+      addCardPopup.renderLoading("Save");
+    });
+}
+
+function handleDeleteButton(card) {
+  deleteImageModal.open();
+  deleteImageModal.handleDelete(() => {
+    deleteImageModal.renderLoading("Saving...");
+    api
+      .deleteCard(card.id)
+      .then(() => {
+        card.deleteCard();
+      })
+      .catch((err) => {
+        console.log(`Unable to process request, ${err}`);
+      })
+      .finally(() => {
+        deleteImageModal.close();
+        deleteImageModal.renderLoading("Yes");
+      });
+  });
+}
+
+function handleLikeButton(card) {
+  if (card.isLiked) {
+    api
+      .deleteLike(card.id)
+      .then(() => {
+        card.handleIsLiked(false);
+      })
+      .catch((err) => {
+        console.log(`Unable to process request, ${err}`);
+      });
+  }
+  if (!card.isLiked) {
+    api
+      .likeCard(card.id)
+      .then(() => {
+        card.handleIsLiked(true);
+      })
+      .catch((err) => {
+        console.log(`Unable to process request, ${err}`);
+      });
+  }
+}
 
-// Event Handlers
-function fillEditProfileForm() {
-  profileTitleInput.value = profileTitle.textContent;
-  profileDescriptionInput.value = profileDescription.textContent;
+function handleChangeProfilePicture(inputUrl) {
+  profileImageModal.renderLoading("Loading...");
+  api
+    .updateProfileImage(inputUrl)
+    .then((res) => {
+      userInfo.setProfileImage(res);
+      profileImageModal.close();
+      profileImageForm.resetValidation();
+      profileImageValidator.disableSubmit();
+    })
+    .catch((err) => {
+      console.log(`Unable to process request, ${err}`);
+    })
+    .finally(() => {
+      profileImageModal.renderLoading("Save");
+    });
 }
 
-// Event Listeners
+//ProfileEditButtonEventListeners
 profileEditButton.addEventListener("click", () => {
   profileEditValidator.hideInputError(profileEditForm);
-  fillEditProfileForm();
-  profileEditPopup.open();
+  const { name, about } = userInfo.getUserInfo();
+  profileTitleInput.value = name;
+  profileDescriptionInput.value = about;
+  profileEditModal.open();
+  profileEditValidator.resetValidation();
 });
-
+//ProfileEditImageEventListeners
+profileEditImage.addEventListener("click", () => {
+  profileEditValidator.hideInputError(profileImageForm);
+  const { url } = userInfo.getUserInfo();
+});
+//Add Card Button Event Listeners
 addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
+  addCardValidator.resetValidation();
```

Comment:
Please, delete it

Method `resetValidation` is for clearing possible validation errors, when you open the **profile** form, because you insert **valid** data into its inputs. But when you add a new card, you don't have any errors after submitting. You need to call `disableButton` there.